Mathlib/Analysis/Normed/Algebra/TrivSqZeroExt.lean

```diff
@@ -141,7 +141,7 @@ variable [Field 𝕜] [CharZero 𝕜] [CommRing R] [AddCommGroup M]
   [Algebra 𝕜 R] [Module 𝕜 M] [Module R M] [Module Rᵐᵒᵖ M]
   [IsCentralScalar R M] [IsScalarTower 𝕜 R M]
   [TopologicalSpace R] [TopologicalSpace M]
-  [TopologicalRing R] [TopologicalAddGroup M] [ContinuousSMul R M] [ContinuousSMul Rᵐᵒᵖ M]
+  [IsTopologicalRing R] [TopologicalAddGroup M] [ContinuousSMul R M] [ContinuousSMul Rᵐᵒᵖ M]
```

Comment:
Everything on this line is Prop-valued; what would you propose the rule is for `Is` vs not?

Reply:
The topology people are sometimes funny about the `Is` convention (because they have some other convention involving e.g. `IsCompact` and also `CompactSpace`) but the algebra people aren't, so I propose `IsTopologicalAddGroup` and leaving the rest alone. What do you think?

Reply:
I have now changed this to `IsTopologicalAddGroup`. I would argue that `Continuous` is a special case, because instead of an `IsContinuous` predicate we typically use the bundled `Continuous` typeclass. I would happily change everything to `Is` on that line but there has been some pushback from people with what I believe are largely spurious objections (e.g. "it unnecessarily makes things a bit longer"). I felt like the community was ready for `IsTopologicalRing` though, nobody really seems to object when I suggest it.